intelmq/bots/collectors/api/REQUIREMENTS.txt

```diff
@@ -1 +1 @@
-tornado>=4.5.2
+tornado>=5.1
```

Comment:
Is this because you only tested it with this version or do the changes require the new version?

Reply:
both, changes are using the latest tornado syntax and only tested for that version

Reply:
Interesting, I successfully tested it with 4.5.3